code/datums/outfits/ert/iac.dm

```diff
@@ -93,5 +95,7 @@
 		/obj/item/device/healthanalyzer = 1,
 		/obj/item/storage/firstaid/adv = 2,
 		/obj/item/storage/firstaid/o2 = 1,
-		/obj/item/storage/box/syringes = 1
+		/obj/item/storage/box/syringes = 1,
+		/obj/item/gun/energy/pistol = 1,
+		/obj/item/clothing/accessory/holster/armpit = 1
```

Comment:
A holster won't help them since they already get a black webbing vest as an accessory.

Reply:
It gives them the option to take the vest off.